### ISSUE TITLE
Adding to Supervision Tree in Phoenix

### ISSUE THREAD

**Issue by kjrocker** [elixir-ecto/myxql#84]
After registering `Ecto.Adapters.MyXQL` as the adapter, fetching/compiling dependencies, etc, I receive the following error: 

`(EXIT) shutdown: failed to start child: DBConnection.ConnectionPool`
`(EXIT) no process: the process is not alive or there's no process currently associated with the given name, possibly because its application isn't started`

I see that I should add MyXQL to the supervision tree. I'm using Phoenix, and wondering where I should put the supervision to get things working. Just registering `:myxql` as an application in `mix.exs` didn't work.

TLDR: Where/how do I register the process in an already existing Phoenix application?

**Comment by josevalim** [comment]
Hi @kjrocker, you shouldn't add MyXQL to the supervision tree, Ecto does it for you. If you are using Phoenix, do `mix phx.new --database mysql` for an example on how everything is setup. If you still have questions, it is best to ask them on the ElixirForum where more people will be available to share their experiences and/or help you. thanks!
